### ISSUE TITLE
pnpm included in build script even if pnpm not chosen as package manager of choice.

### ISSUE THREAD

**Issue by dsandor** [single-spa/create-single-spa#329]
When creating a utility module with no framework and yarn package manager I would expect the build script to use yarn or at least include the pnpm dependency in the devModules section of the package.json template.

For reference, here is the build script with pnpm used to build with concurrently: [util-module.package.json#L6](https://github.com/single-spa/create-single-spa/blob/main/packages/generator-single-spa/src/util-module/templates/util-module.package.json#L6)

Suggested solution: Either remove pnpm from the build script or include pnpm in the devDependencies for this package.json template.

**Example Error**

```
$ concurrently pnpm:build:*
[build:webpack] /bin/sh: pnpm: not found
[build:webpack] pnpm run build:webpack exited with code 127
error Command failed with exit code 1.
```

I did add pnpm into dev dependencies to fix this issue in my generated code. Not sure which direction you wanted to go with the solution.

**Example after adding pnpm to devDependencies**

```
❯ yarn add -D pnpm
yarn add v1.22.11
warning package.json: No license field
warning @foo/cross-app-utility: No license field
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
warning @foo/cross-app-utility: No license field
success Saved 1 new dependency.
info Direct dependencies
└─ pnpm@6.15.0
info All dependencies
└─ pnpm@6.15.0
$ husky install
husky - Git hooks installed
✨  Done in 1.93s.
❯ yarn build
yarn run v1.22.11
warning package.json: No license field
$ concurrently pnpm:build:*
[build:webpack]
[build:webpack] > @foo/cross-app-utility@ build:webpack /Users/dsandor/Projects/cross-app-utility
[build:webpack] > webpack --mode=production
[build:webpack]
[build:webpack] asset nt-cross-app-utility.js 1.34 KiB [emitted] [minimized] (name: main) 1 related asset
[build:webpack] runtime modules 740 bytes 5 modules
[build:webpack] cacheable modules 3.39 KiB
[build:webpack]   modules by path ./node_modules/systemjs-webpack-interop/auto-public-path/*.js 935 bytes
[build:webpack]     ./node_modules/systemjs-webpack-interop/auto-public-path/1.js 89 bytes [built] [code generated]
[build:webpack]     ./node_modules/systemjs-webpack-interop/auto-public-path/auto-public-path.js 846 bytes [built] [code generated]
[build:webpack]   ./src/nt-cross-app-utility.js 117 bytes [built] [code generated]
[build:webpack]   ./node_modules/systemjs-webpack-interop/public-path.js 2.36 KiB [built] [code generated]
[build:webpack] webpack 5.52.1 compiled successfully in 336 ms
[build:webpack] pnpm run build:webpack exited with code 0
✨  Done in 1.17s.
```

Thanks,
David Sandor

**Comment by joeldenning** [comment]
Thanks for the bug report, I've created #333 which fixes this.

**Comment by joeldenning** [comment]
This is released in generator-single-spa@4.1.1 and create-single-spa@4.1.1

See https://github.com/single-spa/create-single-spa/blob/main/packages/generator-single-spa/CHANGELOG.md#411
